docs/tutorials/langchainpy.md

```diff
@@ -68,7 +68,7 @@ The next thing is to send the question and the relevant parts of the docs to the
 ```python
 from langchain.chains import RetrievalQA
 qachain=RetrievalQA.from_chain_type(ollama, retriever=vectorstore.as_retriever())
-qachain.invoke({"query": question})
+print(qachain.invoke({"query": question})['result'])
```

Comment:
Based on other tutorials and demos in the Ollama repository it would be better to showcase storing the result in a variable and print that instead.

```suggestion
res = qachain.invoke({"query": question})
print(res['result'])
```